lib/primer/classify/utilities.yml

```diff
@@ -1269,3 +1269,13 @@
   - hide-lg
   :xl:
   - hide-xl
+:height:
+  :fit:
+  - height-fit
+  :full:
+  - height-full
+:width:
+  :fit:
+  - width-fit
+  :full:
+  - width-full
```

Comment:
did you add those manually? I think you are supposed to add the keys to https://github.com/primer/view_components/blob/ce6e757ca91729f201faeee641ef7967568ba5fa/lib/tasks/utilities.rake#L11-L21 and run the task :)